### ISSUE TITLE
v0.9.3: Vision: Update Tutorial on using multi-dimensional numpy array 

### ISSUE THREAD

**Issue by andrewferlitsch** [andrewferlitsch/Gap#76]
The tutorial should be updated to include two more examples, based on the Tensorflow MNIST training set:

1. Input can be a multi-dimensional numpy array, where first dimension are the images - shape = (55000, 784)

2. Data type of raw pixel data is flat, then it is not normalized (assumed to be normalized).

**Comment by andrewferlitsch** [comment]
done
